### PR TITLE
geos: fetch source code from GitHub

### PR DESCRIPTION
**Pull request by imincik**
## Description of changes

Fetch source code directly from GitHub rather than from OSGeo download server.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by imincik** [comment]
PR checked manually using following command:

```
./result/bin/geosop --explode --format wkt polygonize -a "MULTILINESTRING ((200 100, 100 100, 200 200), (200 200, 200 100), (200 200, 300 100, 200 100))"
POLYGON ((200 100, 100 100, 200 200, 200 100))
POLYGON ((200 100, 200 200, 300 100, 200 100))
```

**Comment by sikmir** [comment]
LGTM, but it would be better to combine it with upgrade to 3.12.0 (perhaps https://github.com/NixOS/nixpkgs/pull/240229), since it rebuilds >100 packages.

**Comment by imincik** [comment]
> LGTM, but it would be better to combine it with upgrade to 3.12.0 (perhaps #240229), since it rebuilds >100 packages.

Yes, that would be ideal.

**Comment by jtojnar** [comment]
Could you please include explanation why this is desirable in the commit message?

**Comment by imincik** [comment]
> Could you please include explanation why this is desirable in the commit message?

Done. 

Why at all people fetch source from pypi if the official source is on GH ? Pypi often contains some build trash files and some other random files. Is there any policy for that ?

**Comment by jtojnar** [comment]
Official Git repo is on GitHub but the download page still promotes osgeo: https://libgeos.org/usage/download/

I am not aware of any specific policy but generally, it is a good idea to follow the upstream instructions as closely as possible when building the package to minimize divergences. In the end, it is up to the individual maintainers.

**Comment by imincik** [comment]
>  In the end, it is up to the individual maintainers.

I am the member of the [geospatial team](https://github.com/NixOS/nixpkgs/blob/2d3a5c7ddb1970199c04a01911335c5960b94913/maintainers/team-list.nix#L324) which maintains this package.

**Comment by imincik** [comment]
OK, let's leave this change for now. @jtojnar , thanks for feedback.

**Comment by sikmir** [comment]
https://github.com/NixOS/nixpkgs/pull/268130 is a good place for it.

**Comment by imincik** [comment]
> #268130 is a good place for it.

I think, I have at least one more reason to fetch source from GitHub - tools like `github:Mic92/nix-update` don't work with `download.osgeo.org`. 

@sikmir , do you think we should add this change to #268130 to avoid another big rebuild ? I spent many hours of running nixpkgs-review for #268130 and I want to get it merged until it gets out of sync with master. Can we leave this change for next round of GEOS update ?

**Comment by jtojnar** [comment]
> I think, I have at least one more reason to fetch source from GitHub - tools like `github:Mic92/nix-update` don't work with `download.osgeo.org`.

The standard option is [`passthru.updateScript`](https://nixos.org/manual/nixpkgs/stable/#var-passthru-updateScript). You can easily use `pkgs.directoryListingUpdater` for this.

But in the end, the acceptability of GitHub-generated tarballs by upstream is what matters. I have encountered several projects that did not want to support packages built from anything other than the official tarballs but, if GEOS team does not mind, using GitHub generated tarballs will be fine as well.

**Comment by imincik** [comment]
> The standard option is [`passthru.updateScript`](https://nixos.org/manual/nixpkgs/stable/#var-passthru-updateScript). You can easily use `pkgs.directoryListingUpdater` for this.

Thanks, I didn't know about this.
 
> But in the end, the acceptability of GitHub-generated tarballs by upstream is what matters. I have encountered several projects that did not want to support packages built from anything other than the official tarballs but, if GEOS team does not mind, using GitHub generated tarballs will be fine as well.

I think the main purpose of OSGeo Download server is just to collect and preserve all main geospatial software tarballs. And even now, we don't fetch all geospatial packages from there - for example [GDAL](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/gdal/default.nix) or [PROJ](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/proj/default.nix) which are core geo libs are downloaded from GitHub.

Actually, there are only few geo packages using `download.osgeo.org`

```
git grep "download.osgeo"
pkgs/development/libraries/geos/3.9.nix:    url = "https://download.osgeo.org/geos/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
pkgs/development/libraries/geos/default.nix:    url = "https://download.osgeo.org/geos/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
pkgs/development/libraries/libLAS/default.nix:    url = "https://download.osgeo.org/liblas/libLAS-${version}.tar.bz2";
pkgs/development/libraries/shapelib/default.nix:    url = "https://download.osgeo.org/shapelib/shapelib-${version}.tar.gz";
pkgs/servers/sql/postgresql/ext/postgis.nix:    url = "https://download.osgeo.org/postgis/source/postgis-${version}.tar.gz";
```

**Comment by Luflosi** [comment]
I feel after the xz situation, fetching git tags directly from the source forge has gained some new importance.

**Comment by imincik** [comment]
> I feel after the xz situation, fetching git tags directly from the source forge has gained some new importance.

Yes, I still want this change to get in.

**Comment by Luflosi** [comment]
Could you please fix the conflict?

**Comment by imincik** [comment]
PR is rebased now. @NixOS/geospatial please review .

File: pkgs/development/libraries/geos/default.nix [hunk 1]
```diff
@@ -12,9 +12,11 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "geos";
   version = "3.13.0";
 
-  src = fetchurl {
-    url = "https://download.osgeo.org/geos/geos-${finalAttrs.version}.tar.bz2";
-    hash = "sha256-R+yD/zNNZyueRCZpXxXabmNoJEIUlx+r84b/jvbfOeQ=";
+  src = fetchFromGitHub {
+    owner = "libgeos";
+    repo = "geos";
+    rev = "${finalAttrs.version}";
```
Comment (Luflosi) [inline pkgs/development/libraries/geos/default.nix:18]:
```suggestion
    tag = finalAttrs.version;
```

**Review by Luflosi — APPROVED** [review]
LGTM.

**Comment by imincik** [comment]
> LGTM.

@Luflosi , thank you very much for review.

**Review by sikmir — APPROVED** [review]

**Comment by imincik** [comment]
Result of `nixpkgs-review pr 255970` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>26 packages marked as broken and skipped:</summary>
  <ul>
    <li>chatgpt-retrieval-plugin</li>
    <li>chatgpt-retrieval-plugin.dist</li>
    <li>cura</li>
    <li>khoj</li>
    <li>khoj.dist</li>
    <li>python311Packages.python-mapnik</li>
    <li>python311Packages.python-mapnik.dist</li>
    <li>python311Packages.rio-tiler</li>
    <li>python311Packages.rio-tiler.dist</li>
    <li>python311Packages.uranium</li>
    <li>python311Packages.worldengine</li>
    <li>python311Packages.worldengine.dist</li>
    <li>python311Packages.wrf-python</li>
    <li>python311Packages.wrf-python.dist</li>
    <li>python312Packages.python-mapnik</li>
    <li>python312Packages.python-mapnik.dist</li>
    <li>python312Packages.rio-tiler</li>
    <li>python312Packages.rio-tiler.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.uranium</li>
    <li>python312Packages.worldengine</li>
    <li>python312Packages.worldengine.dist</li>
    <li>python312Packages.wrf-python</li>
    <li>python312Packages.wrf-python.dist</li>
    <li>t-rex</li>
  </ul>
</details>
<details>
  <summary>25 packages failed to build:</summary>
  <ul>
    <li>apacheHttpdPackages.mod_tile</li>
    <li>netbox</li>
    <li>paperless-ngx</li>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python311Packages.netbox-bgp</li>
    <li>python311Packages.netbox-bgp.dist</li>
    <li>python311Packages.netbox-documents</li>
    <li>python311Packages.netbox-documents.dist</li>
    <li>python311Packages.netbox-plugin-prometheus-sd</li>
    <li>python311Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python311Packages.netbox-reorder-rack</li>
    <li>python311Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.netbox-bgp</li>
    <li>python312Packages.netbox-bgp.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.netbox-plugin-prometheus-sd</li>
    <li>python312Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python312Packages.netbox-reorder-rack</li>
    <li>python312Packages.netbox-reorder-rack.dist</li>
    <li>python312Packages.opensfm</li>
    <li>python312Packages.opensfm.dist</li>
  </ul>
</details>
<details>
  <summary>338 packages built:</summary>
  <ul>
    <li>azure-cli-extensions.rdbms-connect</li>
    <li>azure-cli-extensions.rdbms-connect.dist</li>
    <li>cloudcompare</li>
    <li>easyocr (python312Packages.easyocr)</li>
    <li>easyocr.dist (python312Packages.easyocr.dist)</li>
    <li>entwine</li>
    <li>gdal (python312Packages.gdal)</li>
    <li>gdalMinimal</li>
    <li>geos</li>
    <li>gmt</li>
    <li>gplates</li>
    <li>grass</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>imposm</li>
    <li>inkscape-extensions.inkstitch</li>
    <li>kicadAddons.kikit</li>
    <li>kicadAddons.kikit-library</li>
    <li>kikit</li>
    <li>kikit.dist</li>
    <li>librasterlite2</li>
    <li>librttopo</li>
    <li>librttopo.dev</li>
    <li>libspatialite</li>
    <li>libspatialite.dev</li>
    <li>litestar</li>
    <li>litestar.dist</li>
    <li>mapcache</li>
    <li>mapnik</li>
    <li>mapproxy</li>
    <li>mapproxy.dist</li>
    <li>mapserver</li>
    <li>merkaartor</li>
    <li>mokuro</li>
    <li>mokuro.dist</li>
    <li>mysql-workbench</li>
    <li>nominatim</li>
    <li>octavePackages.mapping</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openorienteering-mapper</li>
    <li>osm2pgsql</li>
    <li>osmscout-server</li>
    <li>otb</li>
    <li>overturemaps</li>
    <li>overturemaps.dist</li>
    <li>paraview</li>
    <li>pdal</li>
    <li>perl538Packages.Tirex</li>
    <li>perl538Packages.Tirex.devdoc</li>
    <li>perl540Packages.Tirex</li>
    <li>perl540Packages.Tirex.devdoc</li>
    <li>pgadmin4</li>
    <li>pgadmin4-desktopmode</li>
    <li>pgadmin4-desktopmode.dist</li>
    <li>pgadmin4.dist</li>
    <li>pgcli (python312Packages.pgcli)</li>
    <li>pgcli.dist (python312Packages.pgcli.dist)</li>
    <li>postgresql13JitPackages.postgis</li>
    <li>postgresql13JitPackages.postgis.doc</li>
    <li>postgresql13Packages.postgis</li>
    <li>postgresql13Packages.postgis.doc</li>
    <li>postgresql14JitPackages.postgis</li>
    <li>postgresql14JitPackages.postgis.doc</li>
    <li>postgresql14Packages.postgis</li>
    <li>postgresql14Packages.postgis.doc</li>
    <li>postgresql15JitPackages.postgis</li>
    <li>postgresql15JitPackages.postgis.doc</li>
    <li>postgresql15Packages.postgis</li>
    <li>postgresql15Packages.postgis.doc</li>
    <li>postgresql16JitPackages.postgis</li>
    <li>postgresql16JitPackages.postgis.doc</li>
    <li>postgresql16Packages.postgis</li>
    <li>postgresql16Packages.postgis.doc</li>
    <li>postgresql17JitPackages.postgis</li>
    <li>postgresql17JitPackages.postgis.doc</li>
    <li>postgresql17Packages.postgis</li>
    <li>postgresql17Packages.postgis.doc</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.asf-search</li>
    <li>python311Packages.asf-search.dist</li>
    <li>python311Packages.avwx-engine</li>
    <li>python311Packages.avwx-engine.dist</li>
    <li>python311Packages.basemap</li>
    <li>python311Packages.basemap.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.cartopy</li>
    <li>python311Packages.cartopy.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.django-bootstrap4</li>
    <li>python311Packages.django-bootstrap4.dist</li>
    <li>python311Packages.django-bootstrap5</li>
    <li>python311Packages.django-bootstrap5.dist</li>
    <li>python311Packages.django-pgactivity</li>
    <li>python311Packages.django-pgactivity.dist</li>
    <li>python311Packages.django-pglock</li>
    <li>python311Packages.django-pglock.dist</li>
    <li>python311Packages.django-tenants</li>
    <li>python311Packages.django-tenants.dist</li>
    <li>python311Packages.drf-extra-fields</li>
    <li>python311Packages.drf-extra-fields.dist</li>
    <li>python311Packages.easyocr</li>
    <li>python311Packages.easyocr.dist</li>
    <li>python311Packages.fiona</li>
    <li>python311Packages.fiona.dist</li>
    <li>python311Packages.flask-admin</li>
    <li>python311Packages.flask-admin.dist</li>
    <li>python311Packages.folium</li>
    <li>python311Packages.folium.dist</li>
    <li>python311Packages.gdal</li>
    <li>python311Packages.geoalchemy2</li>
    <li>python311Packages.geoalchemy2.dist</li>
    <li>python311Packages.geoarrow-pandas</li>
    <li>python311Packages.geoarrow-pandas.dist</li>
    <li>python311Packages.geoarrow-pyarrow</li>
    <li>python311Packages.geoarrow-pyarrow.dist</li>
    <li>python311Packages.geodatasets</li>
    <li>python311Packages.geodatasets.dist</li>
    <li>python311Packages.geopandas</li>
    <li>python311Packages.geopandas.dist</li>
    <li>python311Packages.geoparquet</li>
    <li>python311Packages.geoparquet.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.harlequin-postgres</li>
    <li>python311Packages.harlequin-postgres.dist</li>
    <li>python311Packages.ifcopenshell</li>
    <li>python311Packages.inequality</li>
    <li>python311Packages.inequality.dist</li>
    <li>python311Packages.labelbox</li>
    <li>python311Packages.labelbox.dist</li>
    <li>python311Packages.libpysal</li>
    <li>python311Packages.libpysal.dist</li>
    <li>python311Packages.llama-index-vector-stores-postgres</li>
    <li>python311Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python311Packages.mapclassify</li>
    <li>python311Packages.mapclassify.dist</li>
    <li>python311Packages.mayim</li>
    <li>python311Packages.mayim.dist</li>
    <li>python311Packages.momepy</li>
    <li>python311Packages.momepy.dist</li>
    <li>python311Packages.morecantile</li>
    <li>python311Packages.morecantile.dist</li>
    <li>python311Packages.msticpy</li>
    <li>python311Packages.msticpy.dist</li>
    <li>python311Packages.opensfm</li>
    <li>python311Packages.opensfm.dist</li>
    <li>python311Packages.osmnx</li>
    <li>python311Packages.osmnx.dist</li>
    <li>python311Packages.pgcli</li>
    <li>python311Packages.pgcli.dist</li>
    <li>python311Packages.pgspecial</li>
    <li>python311Packages.pgspecial.dist</li>
    <li>python311Packages.pgvector</li>
    <li>python311Packages.pgvector.dist</li>
    <li>python311Packages.plotnine</li>
    <li>python311Packages.plotnine.dist</li>
    <li>python311Packages.private-gpt</li>
    <li>python311Packages.private-gpt.dist</li>
    <li>python311Packages.psycopg</li>
    <li>python311Packages.psycopg.dist</li>
    <li>python311Packages.psycopg.doc</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.pygmt</li>
    <li>python311Packages.pygmt.dist</li>
    <li>python311Packages.pyogrio</li>
    <li>python311Packages.pyogrio.dist</li>
    <li>python311Packages.pyosmium</li>
    <li>python311Packages.pyosmium.dist</li>
    <li>python311Packages.pyproj</li>
    <li>python311Packages.pyproj.dist</li>
    <li>python311Packages.pytest-postgresql</li>
    <li>python311Packages.pytest-postgresql.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.rapidocr-onnxruntime</li>
    <li>python311Packages.rapidocr-onnxruntime.dist</li>
    <li>python311Packages.rasterio</li>
    <li>python311Packages.rasterio.dist</li>
    <li>python311Packages.rioxarray</li>
    <li>python311Packages.rioxarray.dist</li>
    <li>python311Packages.shapely</li>
    <li>python311Packages.shapely.dist</li>
    <li>python311Packages.shimmy</li>
    <li>python311Packages.shimmy.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python311Packages.visions</li>
    <li>python311Packages.visions.dist</li>
    <li>python311Packages.wktutils</li>
    <li>python311Packages.wktutils.dist</li>
    <li>python311Packages.ydata-profiling</li>
    <li>python311Packages.ydata-profiling.dist</li>
    <li>python312Packages.aplpy</li>
    <li>python312Packages.aplpy.dist</li>
    <li>python312Packages.asf-search</li>
    <li>python312Packages.asf-search.dist</li>
    <li>python312Packages.avwx-engine</li>
    <li>python312Packages.avwx-engine.dist</li>
    <li>python312Packages.basemap</li>
    <li>python312Packages.basemap.dist</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.cartopy</li>
    <li>python312Packages.cartopy.dist</li>
    <li>python312Packages.cf-xarray</li>
    <li>python312Packages.cf-xarray.dist</li>
    <li>python312Packages.django-bootstrap4</li>
    <li>python312Packages.django-bootstrap4.dist</li>
    <li>python312Packages.django-bootstrap5</li>
    <li>python312Packages.django-bootstrap5.dist</li>
    <li>python312Packages.django-pgactivity</li>
    <li>python312Packages.django-pgactivity.dist</li>
    <li>python312Packages.django-pglock</li>
    <li>python312Packages.django-pglock.dist</li>
    <li>python312Packages.django-tenants</li>
    <li>python312Packages.django-tenants.dist</li>
    <li>python312Packages.drf-extra-fields</li>
    <li>python312Packages.drf-extra-fields.dist</li>
    <li>python312Packages.fiona</li>
    <li>python312Packages.fiona.dist</li>
    <li>python312Packages.flask-admin</li>
    <li>python312Packages.flask-admin.dist</li>
    <li>python312Packages.folium</li>
    <li>python312Packages.folium.dist</li>
    <li>python312Packages.geoalchemy2</li>
    <li>python312Packages.geoalchemy2.dist</li>
    <li>python312Packages.geoarrow-pandas</li>
    <li>python312Packages.geoarrow-pandas.dist</li>
    <li>python312Packages.geoarrow-pyarrow</li>
    <li>python312Packages.geoarrow-pyarrow.dist</li>
    <li>python312Packages.geodatasets</li>
    <li>python312Packages.geodatasets.dist</li>
    <li>python312Packages.geopandas</li>
    <li>python312Packages.geopandas.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.glueviz</li>
    <li>python312Packages.glueviz.dist</li>
    <li>python312Packages.harlequin-postgres</li>
    <li>python312Packages.harlequin-postgres.dist</li>
    <li>python312Packages.ifcopenshell</li>
    <li>python312Packages.inequality</li>
    <li>python312Packages.inequality.dist</li>
    <li>python312Packages.labelbox</li>
    <li>python312Packages.labelbox.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph.dist</li>
    <li>python312Packages.libpysal</li>
    <li>python312Packages.libpysal.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.mapclassify</li>
    <li>python312Packages.mapclassify.dist</li>
    <li>python312Packages.mayim</li>
    <li>python312Packages.mayim.dist</li>
    <li>python312Packages.momepy</li>
    <li>python312Packages.momepy.dist</li>
    <li>python312Packages.morecantile</li>
    <li>python312Packages.morecantile.dist</li>
    <li>python312Packages.msticpy</li>
    <li>python312Packages.msticpy.dist</li>
    <li>python312Packages.osmnx</li>
    <li>python312Packages.osmnx.dist</li>
    <li>python312Packages.pgspecial</li>
    <li>python312Packages.pgspecial.dist</li>
    <li>python312Packages.pgvector</li>
    <li>python312Packages.pgvector.dist</li>
    <li>python312Packages.plotnine</li>
    <li>python312Packages.plotnine.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.psycopg</li>
    <li>python312Packages.psycopg.dist</li>
    <li>python312Packages.psycopg.doc</li>
    <li>python312Packages.pvextractor</li>
    <li>python312Packages.pvextractor.dist</li>
    <li>python312Packages.pygmt</li>
    <li>python312Packages.pygmt.dist</li>
    <li>python312Packages.pyogrio</li>
    <li>python312Packages.pyogrio.dist</li>
    <li>python312Packages.pyosmium</li>
    <li>python312Packages.pyosmium.dist</li>
    <li>python312Packages.pyproj</li>
    <li>python312Packages.pyproj.dist</li>
    <li>python312Packages.pytest-postgresql</li>
    <li>python312Packages.pytest-postgresql.dist</li>
    <li>python312Packages.qcodes</li>
    <li>python312Packages.qcodes-contrib-drivers</li>
    <li>python312Packages.qcodes-contrib-drivers.dist</li>
    <li>python312Packages.qcodes.dist</li>
    <li>python312Packages.rapidocr-onnxruntime</li>
    <li>python312Packages.rapidocr-onnxruntime.dist</li>
    <li>python312Packages.rasterio</li>
    <li>python312Packages.rasterio.dist</li>
    <li>python312Packages.rioxarray</li>
    <li>python312Packages.rioxarray.dist</li>
    <li>python312Packages.shapely</li>
    <li>python312Packages.shapely.dist</li>
    <li>python312Packages.spectral-cube</li>
    <li>python312Packages.spectral-cube.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>python312Packages.visions</li>
    <li>python312Packages.visions.dist</li>
    <li>python312Packages.weaviate-client</li>
    <li>python312Packages.weaviate-client.dist</li>
    <li>python312Packages.wktutils</li>
    <li>python312Packages.wktutils.dist</li>
    <li>python312Packages.ydata-profiling</li>
    <li>python312Packages.ydata-profiling.dist</li>
    <li>pytrainer</li>
    <li>pytrainer.dist</li>
    <li>qgis</li>
    <li>qgis-ltr</li>
    <li>qmapshack</li>
    <li>saga</li>
    <li>spatialite_gui</li>
    <li>spatialite_tools</li>
    <li>sumo</li>
    <li>survex</li>
    <li>therion</li>
    <li>tunnelx</li>
    <li>udig</li>
    <li>valhalla</li>
    <li>vpv</li>
  </ul>
</details>

**Review by willcohen — APPROVED** [review]

**Comment by imincik** [comment]
Thanks @willcohen .

**Pull request merged**